libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -829,6 +829,16 @@ singleton AC_PosControl method get_posvelaccel_offset boolean Vector3f'Null Vect
 singleton AC_PosControl method get_vel_target boolean Vector3f'Null
 singleton AC_PosControl method get_accel_target boolean Vector3f'Null
 
+include AC_Sprayer/AC_Sprayer.h depends APM_BUILD_TYPE(APM_BUILD_ArduCopter) || APM_BUILD_TYPE(APM_BUILD_Rover)
```

Comment:
I think maybe this should be dependent upon HAL_SPRAYER_ENABLED rather than the vehicle?